docs/getting-started.md

```diff
@@ -144,6 +144,27 @@ Experiment tracker
 
 By default, Matcha will hide sensitive resource properties. If you need one of these properties, then you can add the `--show-sensitive` flag to your `get` command.
 
+# &#129309; Sharing Resources
+
+You'll notice that a configuration file is create as part of the provisioning process - it's called `matcha.config.json`. This file stores the information necessary for Matcha to link up to your provisioned resources.
+
+To share these resources, you need to add this file to your project repository (GitHub or similar).
+
+> Note: this file does not contain any sensitive information such as passwords or server endpoints.
+
+The other users that you're sharing the resources with will then need to ensure this configuration file exists locally, in the same directory where the file was originally created.
+
+Matcha uses this file to find and pull the provisioned state information, which will allow multiple users to use the same provisioned resources.
+
+<div align="center">
+    <img src="/img/getting-started/shared-state.png" width="400"></img>
+</div>
+
+In order to access the shared state (including secrets) the user needs to do two things:
```

Comment:
I think this section needs to be moved to the top. It's essentially the pre-requisites for sharing state and so it would make sense to me for this to be at the beginning of the section.